Utilities/General/test/BuildFile.xml

```diff
@@ -9,8 +9,3 @@
   <use name="google-benchmark-main"/>
 </bin>
 
-<iftool name="cuda-gcc-support">
```

Comment:
@davidlange6 This is not handled by scram2cmake.py or scram2cmake.pl

Reply:
its valid xml, so I guess scram2cmake needs to be told about it  - is that hard?

Reply:
scram2cmake.py should not see flags, it makes use of SCRAM caches. You might have an old version of SCRAM which does not know about this flag.

Reply:
The error is 
<code>
error in  Utilities/General/test/BuildFile.xml: not well-formed (invalid token): line 15, column 78
</code>
 This is the &&.

Reply:
Changing to  &amp;&amp; removes the error. I will make a pull request with the BuildFile.xml changes for testing.

Reply:
@smuzaffar note I am using scram2cmake.PY so scram is never invoked.

Reply:
A better name for the tool would be buildfile2cmake.py

Reply:
better to change it to ; which will allow scram to build it too.